finance/funding_requests.md

```diff
@@ -0,0 +1,52 @@
+
+Funding requests can be made to the Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 
+
+
+## Travel
```

Comment:
I don't think it need be covered in the scope of this PR (which, per our meeting earlier, is basically about formalizing  existing arrangements), but: what about advances?